app/Http/Controllers/TaskController.php

```diff
@@ -9,7 +9,11 @@ class TaskController extends Controller
 {
     public function index()
     {
-        return view('tasks');
+        $tasks = Task::orderBy('created_at', 'asc')->get();
+
+        return view('tasks', [
+            'tasks' => $tasks,
+        ]);
```

Comment:
Nghiên cứu các cách https://laravel.com/docs/5.5/views#passing-data-to-views e nhé

Reply:
Em tìm hiểu thêm thôi chứ ko sửa lại ở đây chị hè

Reply:
Em thấy cách nào hay hơn thì dùng :D